src/pyird/utils/irdstream.py

```diff
@@ -229,10 +230,65 @@ def flatten(self, trace_path=None, extout='_fl', extin=None, mask=None):
                 hdux = pyf.PrimaryHDU(rsd, header)
                 hdulist = pyf.HDUList([hdux])
                 hdulist.writeto(save_path, overwrite=True)
+
+        if self.info:
+            print('flatten (+ hotpix mask): output extension=', extout)
+
         self.fitsdir = self.anadir
         self.extension = extout
         os.chdir(currentdir)
 
+    def corr_hotpix(self,hotpix_mask, rsd, y0, xmin, xmax, coeff):
```

Comment:
`corr` can be interpreted as `correct` and `correlated` etc?

Reply:
e.g. `apply_hotpixel_mask` is more direct?

Reply:
That's true. I will change it to `apply_hotpixel_mask`.

Reply:
thanks!